webhook-service/handler/handler.go

```diff
@@ -170,13 +171,13 @@ func (th *TaskHandler) onStartedWebhookExecution(keptnHandler sdk.IKeptn, event
 		// if sendFinished is set, we only need to send one started event
 		// the webhook service will then send a correlating .finished event with the aggregated response payloads
 		if err := keptnHandler.SendStartedEvent(event); err != nil {
-			return sdkError(fmt.Sprintf("could not send .started event: %s", err.Error()), err)
+			return sdkError(fmt.Sprintf("could not send .started event: request '%s': %s", webhook.Requests[0], err.Error()), err)
```

Comment:
Can it happen that `webhook.Requests` is empty at this point? Also, is the first of the webhook requests actually tied to this particular error (sending the started event failed)?

Reply:
Actually it shouldn't happen, we are now checking that webhook.Requests are not empty, see this PR below